modules/ROOT/partials/integrations/vue-tech-ref.adoc

```diff
@@ -106,6 +109,7 @@ The editor accepts the following properties:
 
```

Comment:
Can't leave a suggestion sorry, but the code snippet above needs `subs="attributes+"` added to resolve `{productmajorversion}`.

Reply:
done